index.html

```diff
@@ -264,6 +264,55 @@ <h3>
           <p class="arrow2">⌄</p>
         </li>
       </ul>
+      <section class="contactimg">
+        <div class="background-div">
+          <div class="contact-title">
+            <h4>Contact me</h4>
+          </div>
+          <div class="contact-par">
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need to built or a project that needs coding. I'd
+              love to help with it.
+            </p>
+          </div>
+          <div class="form-container">
+            <form action="http://formspree.io/rr155914@gmail.com" method="post">
+              <ul class="list-form">
+                <li class="list4">
+                  <input
+                    type="text"
+                    id="name"
+                    name="user_name"
+                    placeholder="Name"
+                  />
+                </li>
+                <li class="list4">
+                  <input
+                    type="email"
+                    id="mail"
+                    name="user_email"
+                    placeholder="email"
```

Comment:
All input fields should have the `required` attribute to activate the validation on submit. as stated in the project requirement